content/helloworld-fromcftok8s.md

```diff
@@ -1,28 +1,28 @@
-# Moving a Cloud Foundry hello world app to Kubernetes - how hard can it be?
+# Moving a Cloud Foundry Hello World App to Kubernetes - How Hard Can It Be?
 
-Essentially, I've spent most of last year figuring out Cloud Foundry and then telling others about it. And, believe it or not, I sort of fell in love with the simplicity of pushing a changed code base to the platform with one simple command (aka the "cf push experience" (add a link?)). The step from running things locally to running things in the cloud was just one additional file: the Cloud Foundry [manifest](https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html)). It was so low threshold that I actually started not testing changes locally anymore. Instead, I would just push changes to my deployed app immediately. If I had to, I could always hook up a debugger to the deployed app running in Cloud Foundry. 
+Essentially, I've spent most of last year figuring out Cloud Foundry and then telling others about it. And, believe it or not, I sort of fell in love with the simplicity of pushing a changed code base to the platform with one simple command (aka the ["cf push experience"] (https://www.cloudfoundry.org/blog/cloud-foundrys-proven-developer-experience-comes-to-kubernetes-with-cf-for-k8s-1-0/)). The step from running things locally to running things in the cloud was just one additional file: the Cloud Foundry [manifest](https://docs.cloudfoundry.org/devguide/deploy-apps/manifest.html)). It was so low threshold that I actually stopped testing changes locally. Instead, I just pushed changes to my deployed app immediately. If I had to, I could always hook up a debugger to the deployed app running in Cloud Foundry. 
```

Comment:
It might be worth clarifying that you were pushing untested changes to deployed app that was not in production. That might scare some people away...

Reply:
Valid point - added some language to that end.